Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -88,8 +98,15 @@ def render_GET(self, request):
                 request.args["neighbor_level"][0].isdigit():
             neighbor_level = int(request.args["neighbor_level"][0])
 
-        # TODO: Remove dummy return and change to aggregated data calculation
-        return json.dumps({"focus_node": request.args["focus_node"][0],
+        mc_community = self.get_multichain_community()
+        if not mc_community:
+            request.setResponseCode(http.NOT_FOUND)
```

Comment:
I would suggest changing the lines 103 - 104 into a generalization of the return_400() method. You could rename this to return_error(request, http_code, message), so that you can use this method for both the http.NOT_FOUND and http.BAD_REQUEST error returns.